modules/rostests/rosautotest/main.cpp

```diff
@@ -100,6 +110,22 @@ wmain(int argc, wchar_t* argv[])
             WineTest.Run();
         }
 
+        /* Clear the stringstream */
+        ss.str("");
+        ss.clear();
+
+        /* Show the beginning time */
+        ss << "[ROSAUTOTEST] System uptime at start was " << setprecision(2) << fixed;
+        ss << ((float)TestStartTime / 1000) << " seconds" << endl;
+
+        /* Show the time now so that we can see how long it took. */
+        ss << endl
+           << "[ROSAUTOTEST] System uptime at end was " << setprecision(2) << fixed;
+        ss << ((float)GetTickCount()/1000) << " seconds" << endl;
```

Comment:
It may be of interest to store the "GetTickCount()" value in a variable.
Because the function is again called below and there certainly have some ticks that have elapsed between there and below.